install-aws-ofi-nccl.sh

```diff
@@ -27,19 +27,19 @@ git log -1
             --with-nccl=$HOME/nccl/build \
             --with-mpi=/opt/amazon/openmpi
 if [ $? -ne 0 ]; then
-	echo "Configure failed!"
-	exit -1
+    echo "Configure failed!"
```

Comment:
I'm guessing you changed from tabs to spaces, also seems a bit odd to include it in this patch, can you fix it in  the original patch "install-aws-nccl-ofi.sh : return error when any step failed" instead?